src/config/Config.zig

```diff
@@ -5780,6 +5780,27 @@ pub const BackgroundBlur = union(enum) {
     }
 };
 
+/// See window-decoration
+pub const WindowDecoration = enum {
+    true,
```

Comment:
I think that it would be better to call this `client` and have `parseCLI` below translate `true` => `client`.

Reply:
That wouldn't really make sense on macOS, though